Update README.md (#42)
piiano/flows: cli/README.md

- The [flows-cli](./flows-cli.sh) script.
- `realpath` (some older Mac distributions do not contain it - use [homebrew coreutils](https://formulae.brew.sh/formula/coreutils))

#### Docker Desktop configuration
See screenshot below:

> -  Use Virtualization framework should be truned on

Reviewer: Use Virtualization framework - should be truned on

> -  Use Virtualization framework should be truned on

Reviewer 2: ```suggestion
-  `Use Virtualization framework` setting should be enabled
```

> -  File sharing option - select VirtioFS 

Reviewer 2: ```suggestion
-  `File sharing implementation` - select `VirtioFS` 
```

> -  Use Rosetta for x86/amd64 emulation on Apple Silicon - should be turned off 

Reviewer 2: ```suggestion
-  Users of Apple Silicon - disable the usage of Rosetta for x86/amd64 emulation 
```

> -  Users of Apple Silicon - disable the usage of Rosetta for x86/amd64 emulation 

Reviewer 2: ```suggestion
-  For the Apple Silicon users - uncheck  `Use of Rosetta for x86/amd64 emulation on Apple Silicon`
```
